ludwig/api.py

```diff
@@ -655,25 +655,6 @@ def evaluate(
         features_to_load = self.model_definition['input_features'] + \
                            self.model_definition['output_features']
 
-        # todo refactoring: this is needed for image features as we expect all
-        #  inputs to predict to be in memory, but doublecheck
-        num_overrides = override_in_memory_flag(
-            self.model_definition['input_features'],
-            True
-        )
-        if num_overrides > 0:
-            logger.warning(
-                'Using in_memory = False is not supported for Ludwig API.'
-            )
-
-        # preprocessing
-        dataset, training_set_metadata = preprocess_for_prediction(
```

Comment:
Can you clarify why this part is removed?  Are we handling preprocessing somewhere else?

Reply:
The were duplicated code in that section of `evaluate()` method.  You can see it here https://github.com/uber/ludwig/blob/463ce5f511a6fd85b8e5764e487220854b718c27/ludwig/api.py#L658-L696.  I removed the redundant code.

Reply:
Ah gotcha.  Thanks!